File: MIPPA/ClientApp/app/model/session.ts
import { Team } from './team';
import { ScheduleViewModel } from '../viewmodel/schedule/schedule-view-model';

export class Session {
    constructor(
        public sessionId: number,
        public managerId: number,
        public name: string, 
        public format: number, 
        public matchupType: string, 
        public teams: Team[], 
        public schedules: any[]) {}
}

File: MIPPA/Models/Session.cs
using MIPPA.ViewModels;
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Mippa.Models
{
    public class Session
    {
        [Key]
        public int SessionId { get; set; }
        public int ManagerId { get; set; }
        public string Name { get; set; }
        public Format Format { get; set; }
        public MatchupType MatchupType { get; set; }
        public bool ScheduleCreated { get; set; }

        [JsonIgnore]
        public virtual Manager Manager { get; set; }
        public virtual ICollection<Team> Teams { get; set; }
        public virtual ICollection<Schedule> Schedules { get; set; }
        [NotMapped]
        public ScheduleViewModel ScheduleViewModel { get; set; }

        public bool Active { get; set; }
    }
}
